### PR TITLE
[DO NOT MERGE] Remove nxtp everywhere

### LINKED ISSUE #2915: Remove "NXTP"
## Background

We are moving away from all things "NXTP".

User-facing interfaces have already been pruned. This task will cover the rest of the codebase.

## Definition Of Done

No references to NXTP exist in any of the offchain code, packages, etc.

### PR DESCRIPTION
**Pull request by rhlsthrm**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes #2915

### CODE REVIEW

File: ops/mainnet/prod/backend/main.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 terraform {
   backend "s3" {
-    bucket = "nxtp-terraform-mainnet-prod-backend"
+    bucket = "terraform-mainnet-prod-backend"
```
Comment (carlomazzaferro) [inline ops/mainnet/prod/backend/main.tf:3]: This should not change, it result in inconsistent states between what there's deployed and what needs to be redeployed.

One option is migrating state, but that should be done manually and carefully

File: ops/mainnet/prod/core/main.tf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 terraform {
   backend "s3" {
-    bucket = "nxtp-terraform-mainnet-prod-core"
+    bucket = "terraform-mainnet-prod-core"
```
Comment (carlomazzaferro) [inline ops/mainnet/prod/core/main.tf:3]: same here

File: ops/mainnet/prod/core/main.tf [hunk 3]
```diff
@@ -435,7 +435,7 @@ module "ecs" {
   stage                   = var.stage
   environment             = var.environment
   domain                  = var.domain
-  ecs_cluster_name_prefix = "nxtp-ecs"
+  ecs_cluster_name_prefix = "ecs"
```
Comment (carlomazzaferro) [inline ops/mainnet/prod/core/main.tf:438]: would change the name here to be "amarok" or something more descriptive

File: ops/testnet/prod/backend/main.tf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 terraform {
   backend "s3" {
-    bucket = "nxtp-terraform-testnet-prod-backend"
+    bucket = "terraform-testnet-prod-backend"
```
Comment (carlomazzaferro) [inline ops/testnet/prod/backend/main.tf:3]: here

File: ops/testnet/prod/core/main.tf [hunk 5]
```diff
@@ -1,6 +1,6 @@
 terraform {
   backend "s3" {
-    bucket = "nxtp-terraform-testnet-prod-core"
+    bucket = "terraform-testnet-prod-core"
```
Comment (carlomazzaferro) [inline ops/testnet/prod/core/main.tf:3]: anddd here

File: ops/mainnet/prod/backend/main.tf (hunk 1 above)
Comment (rhlsthrm) [reply ops/mainnet/prod/backend/main.tf:3]: Will work on migrating these manually.

File: ops/mainnet/prod/core/main.tf (hunk 3 above)
Comment (rhlsthrm) [reply ops/mainnet/prod/core/main.tf:438]: Don't want to use amarok either. Maybe "bridge-ecs"?

Comment (carlomazzaferro) [reply ops/mainnet/prod/core/main.tf:438]: What would bridge stand for?
But actually, this is just the prefix -- the cluster specification will be baked in into its name, so I think we're good with just ECS.

Comment (rhlsthrm) [reply ops/mainnet/prod/core/main.tf:438]: OK will move back to ecs.

**Review by just-a-node — DISMISSED** [review]
Looks good, didn't see anything missed

**Review by jakekidd — APPROVED** [review]

**Comment by preethamr** [comment]
Not valid anymore. Might redo this with Chimera

**Pull request closed without merging**